src/Flight/FlightController.java

```diff
@@ -26,52 +28,55 @@ public void getFlightInfo() {
     }
 
     public void addFlight() {
-        int indexToReplace = getFirstNull(flights);
-        if (indexToReplace == -1) {
-            System.out.println("Sorry, our flight database is full, please try again later");
-        } else {
-            Flight flightToAdd;
-
-            System.out.println("Please enter flight info, starting with Flight Number: ");
-            int flightNumber = scanner.nextInt();
+        Flight flightToAdd;
 
-            System.out.println("Enter origin airport three letter code (Ex. PHL): ");
-            String originAirport = scanner.next();
+        System.out.println("Please enter flight info, starting with Flight Number: ");
+        int flightNumber = scanner.nextInt();
 
-            System.out.println("Enter destination airport three letter code (Ex. PHL): ");
-            String destinationAirport = scanner.next();
+        System.out.println("Enter origin airport three letter code (Ex. PHL): ");
+        String originAirport = scanner.next();
```

Comment:
nitpick here: usually airport's use the term departure airport instead of origin.